README.md

```diff
@@ -228,28 +228,40 @@ destruction, or when newer block replacing old one.
-    -f, --flatten                 Print flatten json, without DBs Parenthesis
-    -o, --output <FILE>           Specify the output file. If not specified, output goes to stdout
+            -i, --include <EXTRAS>        To include: {aux-val|func}
+            -f, --flatten                 Print flatten json, without DBs Parenthesis
+            -o, --output <FILE>           Specify the output file. If not specified, output goes to stdout
     
     FORMAT_OPTIONS ('resp'):
-    -r, --support-restore         Use the RESTORE command when possible
-    -t, --target-redis-ver <VER>  Specify the target Redis version. Helps determine which commands can
-                                  be applied. Particularly crucial if support-restore being used
-                                  as RESTORE is closely tied to specific RDB versions. If versions not
-                                  aligned the parser will generate higher-level commands instead.
-    -o, --output <FILE>           Specify the output file. If not specified, output goes to stdout
+            -r, --support-restore         Use the RESTORE command when possible
+            -t, --target-redis-ver <VER>  Specify the target Redis version. Helps determine which commands can
+                                          be applied. Particularly crucial if support-restore being used 
+                                          as RESTORE is closely tied to specific RDB versions. If versions not
+                                          aligned the parser will generate higher-level commands instead.
+            -o, --output <FILE>           Specify the output file. If not specified, output goes to stdout
+            -e, --enum-commands           Command enumeration by preceding commands with `echo <cmd-id>'
     
     FORMAT_OPTIONS ('redis'):
-    -r, --support-restore         Use the RESTORE command when possible
-    -t, --target-redis-ver <VER>  Specify the target Redis version
-    -h, --hostname <HOSTNAME>     Specify the server hostname (default: 127.0.0.1)
-    -p, --port <PORT>             Specify the server port (default: 6379)
-    -l, --pipeline-depth <VALUE>  Number of pending commands before blocking for responses
+            -r, --support-restore         Use the RESTORE command when possible
+            -t, --target-redis-ver <VER>  Specify the target Redis version
+            -h, --hostname <HOSTNAME>     Specify the server hostname (default: 127.0.0.1)
+            -p, --port <PORT>             Specify the server port (default: 6379)
+            -l, --pipeline-depth <VALUE>  Number of pending commands before blocking for responses
+            -s, --start-cmd-num <NUM>     Start writing redis from command number
+            -u, --user <USER>             Redis username for authentication
+            -P, --password <PWD>          Redis password for authentication
+            -c, --custom-auth <CMD>       Customized authentication command in case of proxy
```

Comment:
Proxy is a general term. I think this is the common and only use case for custom authentication. 
It helps to understand the purpose of this command.

Reply:
custom authentication can also be needed by some forks of redis or redis-re-implementation of other databases with redis-compatible interface.
custom authentication can even be needed when someone integrates a module that provides such mechanism.
i don't see how "proxy" is a generic term here.
we can elaborate the explanation with a lot of details, or just make it shorter and generic, like "Issue a different command instead of AUTH" or such.